sunpy/io/tests/test_fits.py

```diff
@@ -176,3 +176,11 @@ def test_warn_longkey():
                                'goodkey': 'test'})
     assert 'GOODKEY' in fits.keys()
     assert 'BADLONGKEY' not in fits.keys()
+
+
+def test_read_memmap():
+    # Test that memmap is passed correctly to the FITS reader
+    data, header = sunpy.io._fits.read(AIA_171_IMAGE, memmap=True)[0]
+    # Create memmap array with data and header
+    mm = np.memmap(AIA_171_IMAGE, dtype=data.dtype, mode='r', shape=data.shape)
+    assert isinstance(mm, np.memmap)
```

Comment:
I would suggest looking at this test and asking yourself, what are you testing?

Reply:
Sorry for that, is it good now?

Reply:
Yes, much better.